src/repl.jl

```diff
@@ -241,19 +240,29 @@ function completions(c::DebugCompletionProvider, full, partial)
     frame = c.state.frame
 
     # repl backend completions
-    comps, range, should_complete = REPLCompletions.completions(full, lastindex(partial), moduleof(frame))
-    ret = map(REPLCompletions.completion_text, comps) |> unique!
+    comps1, range1, should_complete1 = REPLCompletions.completions(full, lastindex(partial), moduleof(frame))
+    ret1 = map(REPLCompletions.completion_text, comps1)
 
-    # local completions
-    vars = filter!(locals(frame)) do v
-        # ref: https://github.com/JuliaDebug/JuliaInterpreter.jl/blob/master/src/utils.jl#L365-L370
-        if v.name == Symbol("#self") && (v.value isa Type || sizeof(v.value) == 0)
-            return false
-        else
-            return startswith(string(v.name), partial)
-        end
-    end |> vars -> map(v -> string(v.name), vars)
-    pushfirst!(ret, vars...)
+    ignore_local(v) = v.name == Symbol("#self") && (v.value isa Type || sizeof(v.value) == 0)
+    m = Module()
+    for v in locals(frame)
+        ignore_local(v) && continue
+        Base.eval(m, :($(v.name) = $(v.value)))
```

Comment:
```suggestion
        Base.eval(m, :($(v.name) = $(QuoteNode(v.value))))
```
I guess we need to quote it? Otherwise `Symbol` values etc may cause undef var error here.

Reply:
Thanks, good catch